paper.tex

```diff
@@ -391,67 +400,115 @@ \subsection{RSE Outreach}
+
+We propose a realization strategy for a central institutional RSE department.
+We start by listing different possibilities for funding RSE positions at a research institution.
+Following that, we describe a potential transition pathway, starting from existing structures that have grown in research alliances such as collaborative research centers or also in research departments of an institution.
+This is complemented by discussing the opportunity of outsourcing RSE services and the challenging task of identifying and hiring suitable RSE candidates.
+
+\subsection{Funding possibilities}
+\label{sec:funding}
+
+We see four basic options for financing RSE positions, which we will briefly explain below:
+\begin{enumerate}
+\item ordinary budget positions,
+\item the overhead of externally funded projects,
+\item explicitly requested person-months in externally funded projects, and
+\item dedicated RSE calls.
+\end{enumerate}
+At research institutions, it is important to resolve the conflict between time-limited research funding and the need for permanent positions in order to be competitive with industry. Experience is also an essential component of software engineering, which makes long-term employment indispensable. In principle, pooling of positions and funds makes it possible to finance permanent positions from changing and mixed sources. Nevertheless, convincing the institution to take the corresponding risk of failing to raise external funds may be a very challenging task.
+\begin{enumerate}
+\item It seems natural to allocate ordinary budget positions for RSEs. However, particularly at German universities, it is usually impossible to create completely new budget positions and the only feasible way is to rededicate an existing position after its corresponding holder has left. Depending on the local circumstances at an institution, this can be a cumbersome and time-consuming process for each such position.
+\item Funding organizations additionally allocate a portion of the direct project funds as overhead, which is typically divided between the institution and the applicant. We propose using a small percentage of the overhead agglomorated at the institution to permanently finance central RSE positions. Assuming a third-party funding income of EUR 50 million annually and a 20\% overhead, EUR 100,000 in permanent funding requirements for one person-year would only account for 1\% of this overhead.
+\item In project applications involving the development of research software, corresponding person-months should be applied for to finance RSE tasks.
+      In this way, an applicant can book a fixed number of working hours from the RSE pool and pay for the costs accordingly.
+      This model has been successfully implemented at several UK universities.
+      In order to scale, it needs to be supported by an institutional policy.
+      Larger collaborative projects can apply for funding as part of a central INF project.
```

Comment:
should we explain INF projects? Mine is still called Z03 ....

Reply:
We could exemplify and spell it out: "as part of, for example, an information infrastructure project"